_data/simple-icons.json

```
@@ -9356,8 +9356,8 @@
         },
         {
             "title": "ProtonMail",
-            "hex": "8B89CC",
-            "source": "https://protonmail.com/media-kit"
+            "hex": "9D82FF",
```

I'd prefer to use their theme color #6d4aff due to the gradient being hard to choose.

```suggestion
            "hex": "6D4AFF",
```